packages/Operators/src/PointCloud/DTK_MovingLeastSquareReconstructionOperator_impl.hpp

```diff
@@ -94,15 +94,15 @@ MovingLeastSquareReconstructionOperator(
     // If we are doing kNN support get the number of neighbors.
     if( d_use_knn )
     {
-	DTK_REQUIRE( parameters.isParameter("Search Num Neighbors") );
-	d_knn = parameters.get<int>("Search Num Neighbors");
+	DTK_REQUIRE( parameters.isParameter("Num Neighbors") );
+	d_knn = parameters.get<int>("Num Neighbors");
     }
     
     // Otherwise we are doing the radius search so get the basis radius.
     else
     {
-	DTK_REQUIRE( parameters.isParameter("Search Radius") );
-	d_radius = parameters.get<double>("Search Radius");
+	DTK_REQUIRE( parameters.isParameter("RBF Radius") );
+	d_radius = parameters.get<double>("RBF Radius");
```

Comment:
If I hadn't changed it back it wouldn't have been backwards compatible. This was the same as the previous version